davidagitlen/Tripedia-fe
# Continue form functionality

#### What’s this PR do?

Completes test functionality of all but last form in side menu, incorporates changes of previous merge to master.

#### Where should the reviewer start?

All Form components have been altered. 

#### How should this be manually tested?

Upon going to the home page '/' you should be able to expand and collapse each form except the last and check boxes, observing that they update in the state object provided by the hook of the component. 

#### Any background context you want to provide?

Tried to avoid previous git problems by pulling down updated master and merging locally. 

#### What are the relevant tickets?

See below. 

#### Screenshots (if appropriate)

N/A

#### Questions:

# Implements/Fixes:
* What issue does this close? #33, #34, #35, #36, #37 on Tripedia Project Board/Issues in Backend
